docs/troubleshooting/module-does-not-exist-in-module-map.md

```diff
@@ -0,0 +1,15 @@
+# Could Not Load Module Because It Does Not Exist in the Module Version Map
+
+#### Why This Error Occurred
+
+Sometimes you might want to compose a module from multiple smaller modules using either [iguazu-holocron](https://github.com/americanexpress/holocron/tree/main/packages/iguazu-holocron) or [RenderModule](https://github.com/americanexpress/holocron/blob/main/packages/holocron/docs/api/README.md#rendermodule). However, if you have not added the module you are trying to load to your [module-map](../api/server/Module-Map-Schema), it will fail to load.
```

Comment:
I think this error also occurs if you use `RenderModule` without dispatching `composeModules` in `LoadModuleData` could you please check if that's the case add it here?